src/filter/glitch0r/glitch0r.c

```diff
@@ -0,0 +1,347 @@
+/* glitch0r.c
+ * Copyright (C) 2016 IDENT Software ~ http://identsoft.org
+ * Inspired by the test video on frei0r.dune.org
+ * 
+ * This file is a Frei0r plugin.
+ *
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation; either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 675 Mass Ave, Cambridge, MA 02139, USA.
+ */
+
+#include <stdlib.h>
+#include <assert.h>
+#include <string.h>
+#include <time.h>
+
+#include "frei0r.h"
+
+/* cheap & fast randomizer (by Fukuchi Kentarou) */
+uint32_t randval;
```

Comment:
Please make all private symbols static. Names such as fastrand and fastsrand have a strong chance of collision.